src/pages/publication/NewPublication.tsx

```diff
@@ -80,8 +74,8 @@ const NewPublication: FC = () => {
                 openForm={handleClick}
               />
             </StyledSelectorWrapper>
-            <StyledInfoBox>
-              <header>{t('common:information')}</header>
+            <StyledCard variant="outlined">
+              <FormCardHeading>{t('common:information')}</FormCardHeading>
```

Comment:
Skulle vi bare kalt den for CardHeading? Her er det jo ingen form. Gjelder sikkert flere av komponentene også?

Reply:
Vi diskuterte å fjerne form-prfikset på card-komponentene. Vil helst unngå å bruke "card" pga mulige konflikter, men klarte ikke finne et bra navn.